lib/engine/game/g_1873.rb

```diff
@@ -346,8 +531,28 @@ def price_movement_chart
         ]
       end
 
+      def corporation_view(corporation)
+        if corporation.minor?
+          'independent_mine'
+        elsif public_mine?(corporation)
+          'public_mine'
+        end
+      end
+
+      def status_str(corporation)
+        return if corporation == @mhe
+
+        str = "Maint: #{format_currency(maintenance_costs(corporation))}"
```

Comment:
In the screenshots it seems like there's enough room to say "Maintenance", is there a good reason to keep it shorter?

Reply:
No, not really.